src/services/rooms-services.js

```diff
@@ -1,7 +1,61 @@
-import Rooms from '../constants/rooms-constants';
+import axios from './axios';
+import { isRoomAvailable, getMyBookings } from '../utils/rooms';
+import serverData from '../constants/server-constants';
 
 export default {
-  getRooms() {
-    return Rooms;
+  async getRooms() {
+    const response = await axios.get('/rooms');
+
+    return response.data;
+  },
+  async getAuthGroup({ authId }) {
+    const data = await axios.get(`/user/${authId}`, {
+      profileId: serverData.SERVER_PROFILE_ID,
+      authId,
+      isSecondaryAuth: false,
+      connectionName: '',
+    });
+    return data.data;
+  },
+  async bookRoom({ authId, startHour, endHour, roomId, eventName }) {
+    const response = await axios.post('/book', {
+      user: authId,
+      room: roomId,
+      startTime: startHour,
+      endTime: endHour,
+      name: eventName,
+    });
+
+    return response.data;
+  },
+
+  async bookUpdate({ bookingId, startHour, endHour, roomId }) {
+    const response = await axios.put('/book', {
+      bookingId,
+      room: roomId,
+      startTime: startHour,
+      endTime: endHour,
+    });
+
+    return response.data;
+  },
+
+  async getRoomsAvailability({ when }) {
+    const response = await this.getRooms();
+    return response.map(el => ({
+      id: el.id,
+      name: el.name,
+      floor: el.floor,
```

Comment:
~~This info is already calculated in our middleware~~
Okey, this function do more that just get the availability. Then, we should change the name to reflect what it really does.